lib/vmpooler/graphite.rb

```diff
@@ -35,6 +35,8 @@ def log(path, value)
           socket.close
         end
       end
+    rescue Errno::EADDRNOTAVAIL => e
```

Comment:
Are there any other `Errno` we'd want to catch here? `Errno::EACCES` and `Errno::EADDRINUSE` seem like those are probably fatal errors but would `Errno::EFAULT` ever be non-fatal?

Reply:
Quite possibly. EADDRNOTAVAIL caught my attention because it's what our instance is yelling about now when it cannot get a connection to graphite. I'm not sure when we would encounter the others, but am not opposed to adding if you'd like to see those too.

Reply:
Problem is, I'm not entirely sure which ones would be useful to add, and which would hide a fatal problem. 😆  Maybe it's better to just ship this for now and figure out additional errors to catch later...